src/main/scala/com/mwz/sonar/scala/scoverage/ScoverageSensor.scala

```diff
@@ -32,22 +32,17 @@ import scalariform.ScalaVersion
 import scala.collection.JavaConverters._
 import scala.util.{Failure, Success, Try}
 
-/** Main sensor for importing Scoverage reports to SonarQube */
```

Comment:
I would leave this comment instead of the other, what do you think?

Reply:
Yes, I changed it.